jhub_apps/themes/__init__.py

```diff
@@ -7,16 +7,17 @@
     "logo": LOGO,
     "favicon": FAVICON,
     "primary_color": "#ba18da",
+    "primary_color_light": "#BA18DA10",
     "primary_color_dark": "#9b00ce",
     "secondary_color": "#18817a",
     "secondary_color_dark": "#12635e",
     "accent_color": "#eda61d",
     "accent_color_dark": "#a16d14",
-    "text_color": "#1c1d26",
+    "text_color": "#f00ad1",
     "h1_color": "#0f1015",
     "h2_color": "#0f1015",
-    "navbar_text_color": "#ffffff",
-    "navbar_hover_color": "#20b1a8",
+    "navbar_text_color": "#2E2F33",
+    "navbar_hover_color": "#00000008",
     "navbar_color": "#1c1d26",
```

Comment:
If changing the navbar_text_color to black, the navbar_color should probably be a shade of white?